tests/installation/ipxe_install.pm

```diff
@@ -87,19 +95,46 @@ sub set_bootscript {
         $initrd .= "/boot/$arch/loader/initrd";
     }
 
+    if (get_var('SUT_NETDEVICE') and !is_tumbleweed) {
+        my $interface = get_var('SUT_NETDEVICE');
+        $install .= "?device=$interface ifcfg=$interface=dhcp4 ";
+    }
 
     my $cmdline_extra;
     $cmdline_extra .= " regurl=$regurl " if $regurl;
     $cmdline_extra .= " console=$console " if $console;
 
+    # Support passing both EXTRA_PXE_CMDLINE to bootscripts
+    $cmdline_extra .= get_var('EXTRA_PXE_CMDLINE') . ' ' if get_var('EXTRA_PXE_CMDLINE');
     $cmdline_extra .= " root=/dev/ram0 initrd=initrd textmode=1" if check_var('IPXE_UEFI', '1');
 
     if ($autoyast ne '') {
-        $cmdline_extra .= " autoyast=$autoyast ";
+        $cmdline_extra .= " autoyast=$autoyast sshd=1 sshpassword=$testapi::password ";
     } else {
-        $cmdline_extra .= " sshd=1 vnc=1 VNCPassword=$testapi::password sshpassword=$testapi::password ";    # trigger default VNC installation
+        if (check_var('VIDEOMODE', 'text')) {
+            $cmdline_extra .= " ssh=1 sshpassword=$testapi::password ";    # trigger ssh-text installation
+        }
+        else {
+            $cmdline_extra .= " vnc=1 VNCPassword=$testapi::password ";    # trigger default VNC installation
+        }
+    }
+    $cmdline_extra .= " plymouth.enable=0 ";
+
+    # Extra options for virtualization tests with ipmi backend
+    if (get_var('VIRT_AUTOTEST') || get_var('HANA_PERF')) {
+        $cmdline_extra .= " vga=791 video=1024x768 vt.color=0x07";
```

Comment:
Some options, like ```vga``` is already deprecated.

Reply:
Are these 3 parameters all deprecated? I am not aware of it. I'll do some reseach and tests before removing them.

Reply:
Are these 3 parameters all deprecated? I am not aware of it. I'll do some reseach and tests before removing them.

Reply:
`vga=xx` is deprecated in 32-bit linux and Grub2. `video=xx` and `vt.colar=xx` are still in use.  I removed `vga=791`.